### DISCUSSION TITLE
Slider config for GCCwGTV

### DISCUSSION THREAD

**Discussion by soeoeb** [Nerwyn/universal-remote-card#72] [Q&A]
Hey guys, I am not very competent in reading/writing/understanding code by myself… 😅 I tried setting up the volume slider for my remote card with the help of the README but obviously failed miserably because I am getting following error:

> Logger: homeassistant.components.websocket_api.http.connection
> Quelle: components/websocket_api/commands.py:239
> Integration: Home Assistant WebSocket API (Dokumentation, Probleme)
> Erstmals aufgetreten: 19:01:01 (18 Vorkommnisse)
> Zuletzt protokolliert: 20:29:56
> 
> [140269299191616] Entity media_player.sebs_tv does not support this service.
> [140269206546752] Entity media_player.sebs_tv does not support this service.
> [140269163230784] Entity media_player.sebs_tv does not support this service.
> [140269105542080] Entity media_player.sebs_tv does not support this service.
> Traceback (most recent call last):
>   File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 239, in handle_call_service
>     response = await hass.services.async_call(
>                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/usr/src/homeassistant/homeassistant/core.py", line 2319, in async_call
>     response_data = await coro
>                     ^^^^^^^^^^
>   File "/usr/src/homeassistant/homeassistant/core.py", line 2356, in _execute_service
>     return await target(service_call)
>            ^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 887, in entity_service_call
>     raise HomeAssistantError(
> homeassistant.exceptions.HomeAssistantError: Entity media_player.sebs_tv does not support this service.

Anyone with a working volume slider care to share their insights/config? ☺️

**Comment by Nerwyn** [comment]
```
[140269105542080] Entity media_player.sebs_tv does not support this service.
```
Unfortunately, it looks like your TV does not support the `media_player.volume_set` service, so it can't be used with the slider. You have to use the volume buttons instead.
